README.md

```diff
@@ -36,4 +37,4 @@ https://www.python.org/downloads/
 
 * [Begin your Python Journey](beginning.md)
 * [Have fun with Python Data Structures](datat_structures.md)
```

Comment:
this link does not work, please fix it in the same pr and write in the commit message what other things you fixed.